default.nix

```diff
@@ -234,6 +234,7 @@ let
       ''}
       (load "${doom-emacs}/lisp/doom.el")
       (load "${doom-emacs}/lisp/doom-start.el")
+      (load "${./info-workarounds.el}")
```

Comment:
Any reason to not add this to the `$out/config.el` instead?

This is supposed to be used only for early init of Emacs.

Reply:
`$out/config.el` (stage 4) did not work.

Before putting this line 237 here, I tried putting it instead in all of the following lines: 

- stage 4: lines 220, 221, 222, 223
	in any of these lines, the fix does not go through (it gets overridden)

- stage 5: lines 231, 235, 236
 	in any of these lines, the following error happens: `Symbol's value as variable is void: Info-directory-list`

I can try other places if you have suggestions.

Reply:
Fair enough :+1: .